src/pages/Stake/Index.tsx

```diff
@@ -1,5 +1,3 @@
-import React from "react";
```

Comment:
lol i thought this is required for every .tsx file, looks like it's not

Reply:
ts lint asks that, we can disable it in .tsconfig and .eslintrc files